packages/drafts-realm/Item/8ec1b337-1a53-479e-9bb8-3047e714142c.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Generic names are becoming an issue...

Reply:
Yeah I'll rename - originally I tried putting it in a folder but then some of the creation stuff broke, I assume with relative links.